sites/novascotiajobs/1-job-99.html

```diff
@@ -2,8 +2,8 @@
 layout: default
 title: BC Jobs - 7 Day Free Trial
 landing-background-image: '/assets/img/handshake.jpg'
-permalink: novascotiajobs/7-day-trial-b
-post_url: "https://www.novascotiajobs.ca/coupons/7-day-trial-blue"
+permalink: novascotiajobs/1-job-99
+post_url: "https://www.novascotiajobs.ca/coupons/1-job-99-x1435"
```

Comment:
Why does this `post_url` have quotes around it when albertajobcentre's doesn't?

Reply:
Removed quotes. Did so with rebase and force push. Squashed into `fix permalinks` commit